Modules/_ssl.c

```diff
@@ -4711,9 +4711,8 @@ _ssl_MemoryBIO_read_impl(PySSLMemoryBIO *self, int len)
 
     nbytes = BIO_read(self->bio, PyBytes_AS_STRING(result), len);
     /* There should never be any short reads but check anyway. */
-    if ((nbytes < len) && (_PyBytes_Resize(&result, len) < 0)) {
-        Py_DECREF(result);
-        return NULL;
+    if (nbytes < len) {
+        _PyBytes_Resize(&result, len);
```

Comment:
This operation doesn't have any effect. There should be `nbytes` instead of `len`. Note that `nbytes` can be negative, this denotes an error.